packages/create-react-native-library/src/index.ts

```diff
@@ -17,43 +18,46 @@ const COMMON_FILES = path.resolve(__dirname, '../templates/common');
 const JS_FILES = path.resolve(__dirname, '../templates/js-library');
 const EXPO_FILES = path.resolve(__dirname, '../templates/expo-library');
 const CPP_FILES = path.resolve(__dirname, '../templates/cpp-library');
-const EXAMPLE_FILES = path.resolve(__dirname, '../templates/example');
-const EXAMPLE_TURBO_FILES = path.resolve(
-  __dirname,
-  '../templates/example-turbo'
-);
+const EXAMPLE_FILES = path.resolve(__dirname, '../templates/example-legacy');
+const EXAMPLE_NEW_FILES = path.resolve(__dirname, '../templates/example-new');
 const NATIVE_COMMON_FILES = path.resolve(
   __dirname,
   '../templates/native-common'
 );
 
 const NATIVE_FILES = {
   module_legacy: path.resolve(__dirname, '../templates/native-library-legacy'),
-  module_turbo: path.resolve(__dirname, '../templates/native-library-turbo'),
+  module_new: path.resolve(__dirname, '../templates/native-library-new'),
   module_mixed: path.resolve(__dirname, '../templates/native-library-mixed'),
-  view: path.resolve(__dirname, '../templates/native-view-library'),
+  view_legacy: path.resolve(__dirname, '../templates/native-view-legacy'),
+  view_mixed: path.resolve(__dirname, '../templates/native-view-mixed'),
+  view_new: path.resolve(__dirname, '../templates/native-view-new'),
 };
 
 const JAVA_FILES = {
   module_legacy: path.resolve(__dirname, '../templates/java-library-legacy'),
-  module_turbo: path.resolve(__dirname, '../templates/java-library-turbo'),
+  module_new: path.resolve(__dirname, '../templates/java-library-new'),
   module_mixed: path.resolve(__dirname, '../templates/java-library-mixed'),
-  view: path.resolve(__dirname, '../templates/java-view-library'),
+  view_legacy: path.resolve(__dirname, '../templates/java-view-legacy'),
+  view_mixed: path.resolve(__dirname, '../templates/java-view-mixed-library'),
```

Comment:
I couldn't see this folder under templates and this might be responsible for [this issue](https://github.com/callstack/react-native-builder-bob/pull/273#issuecomment-1241710146)

Reply:
Thank you for checking it. That was an issue